sky/skylet/providers/lambda_cloud/lambda_utils.py

```diff
@@ -2,10 +2,17 @@
 import json
 import os
 import requests
+import time
 from typing import Any, Dict, List, Optional, Tuple
 
+from sky.utils import common_utils
+
 CREDENTIALS_PATH = '~/.lambda_cloud/lambda_keys'
 API_ENDPOINT = 'https://cloud.lambdalabs.com/api/v1'
+# TODO(tian): Determine best backoff factors.
+INITIAL_BACKOFF = 3
```

Comment:
I tried the repro and for the first ~3 minutes, no launch can go through:
```
I 07-24 20:59:38 cloud_vm_ray_backend.py:1732] Launching on Lambda us-west-1
⠧ LaunchingI 07-24 20:59:38 cloud_vm_ray_backend.py:1732] Launching on Lambda us-west-1
⠇ LaunchingI 07-24 20:59:38 cloud_vm_ray_backend.py:1732] Launching on Lambda us-west-1
⠋ LaunchingI 07-24 20:59:38 cloud_vm_ray_backend.py:1732] Launching on Lambda us-west-1
⠹ LaunchingI 07-24 20:59:38 cloud_vm_ray_backend.py:1732] Launching on Lambda us-west-1
⠹ LaunchingI 07-24 21:00:18 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:00:18 cloud_vm_ray_backend.py:1826] Retrying launching in 6.8 seconds.
⠦ LaunchingI 07-24 21:00:32 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:00:32 cloud_vm_ray_backend.py:1826] Retrying launching in 5.1 seconds.
⠦ LaunchingI 07-24 21:00:39 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:00:39 cloud_vm_ray_backend.py:1826] Retrying launching in 6.8 seconds.
⠋ LaunchingI 07-24 21:01:14 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:01:14 cloud_vm_ray_backend.py:1826] Retrying launching in 4.0 seconds.
⠇ LaunchingI 07-24 21:01:24 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:01:24 cloud_vm_ray_backend.py:1826] Retrying launching in 7.1 seconds.
I 07-24 21:01:24 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:01:24 cloud_vm_ray_backend.py:1826] Retrying launching in 14.1 seconds.
⠧ LaunchingI 07-24 21:01:45 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:01:45 cloud_vm_ray_backend.py:1826] Retrying launching in 7.8 seconds.
⠧ LaunchingI 07-24 21:02:00 cloud_vm_ray_backend.py:1793] Retrying due to Lambda API rate limit exceeded.
I 07-24 21:02:00 cloud_vm_ray_backend.py:1826] Retrying launching in 17.2 seconds.
⠹ Launching
```

Should we try increasing initial backoff to say like 10 seconds to lessen the contention?

Reply:
Tested initial = 10s and it performs very well (not a single retry happened when launching 5 clusters simultaneously). I think 10s is a good choice.